app/code/Magento/Bundle/Test/Unit/Model/LinkManagementTest.php

```diff
@@ -610,12 +622,15 @@ static function () {
     public function testAddChild()
     {
         $productLink = $this->getMockBuilder(LinkInterface::class)
-            ->setMethods(['getSku', 'getOptionId', 'getSelectionId'])
+            ->setMethods(['getSku', 'getOptionId', 'getSelectionId', 'getWebSiteId'])
```

Comment:
This method doesn't exist anymore on the Link, please remove it.

Reply:
Done